src/current/v25.2/architecture/replication-layer.md

```diff
@@ -160,6 +160,15 @@ Unlike table data, system ranges use expiration-based leases; expiration-based l
 
 Expiration-based leases are also used temporarily during operations like lease transfers, until the new Raft leader can be fortified based on store liveness, as described in [Leader leases](#leader-leases).
 
+#### Leader‑leaseholder splits
+
+[Epoch-based leases](#epoch-based-leases) are vulnerable to _leader-leaseholder splits_. These can occur when a leaseholder’s Raft log has fallen behind and cannot acquire Raft leadership. Coupled with a [network partition]({% link {{ page.version.version }}/cluster-setup-troubleshooting.md %}#network-partition), this split can cause permanent unavailability of the range if (1) the stale leaseholder continues heartbeating the [liveness range](#liveness-range) to hold its lease but (2) cannot reach the leader to propose writes.
```

Comment:
Replace all smart quotes

```suggestion
[Epoch-based leases](#epoch-based-leases) are vulnerable to _leader-leaseholder splits_. These can occur when a leaseholder's Raft log has fallen behind and cannot acquire Raft leadership. Coupled with a [network partition]({% link {{ page.version.version }}/cluster-setup-troubleshooting.md %}#network-partition), this split can cause permanent unavailability of the range if (1) the stale leaseholder continues heartbeating the [liveness range](#liveness-range) to hold its lease but (2) cannot reach the leader to propose writes.
```

Reply:
I might just be tired at the moment, but "Raft log has fallen behind" might be unclear as to "what" the Raft log has fallen behind.